team.html

```diff
@@ -147,12 +148,12 @@ <h2>Our Team</h2>
               <div class="col-md-6 col-lg-3 text-center" data-aos="fade-down-right">
                 <img src="./assets/img/team-2.jpg" alt="img-team" class="img-fluid rounded-circle mb-3" style="width: 12.5rem;">
                 <p class="fw-bold mb-0">Chief Operating Officer & Co-Founder</p>
-                <p>Felix Pratama, S.T., M.BA (Cand)</p>
```

Comment:
The team's description is obsolete. Please use the new one

Reply:
done, you can check it aagain :D